app/src/main/java/com/android/sample/model/profile/Profile.kt

```diff
@@ -0,0 +1,52 @@
+package com.android.sample.model.profile
+
+data class Profile(
+    val uid: String, // Unique string id for the profile
+    val firstName: String, // First name of the profile
+    val lastName: String, // Last name of the profile
+    val role: Role, // Role of the profile
+    val section: Section, // Section of the profile
+    val academicLevel: AcademicLevel, // Academic level of the profile
+    val schedule: List<List<Int>>, // Weekly schedule of the profile (7 days x 12 slots)
+    val email: String, // Email of the profile
+)
```

Comment:
We should consider storing lessons ID's in each profile, but this should be discussed in group and can be modified in the future!